docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,216 @@
+
+### Don’t Forget to Save Your Work
+Working in the cloud has its advantages. There are some differences though. Unlike working on a desktop where you save your files to a local machine, you must either deploy your changes to your org, or commit them to source control to save your work. Remember to do this before you close the Code Builder tab on your browser so you don’t lose your work.  Throughout the beta, Code Builder environments could be deleted.  All beta environments will be removed before GA.
+
+
+## Let’s Get Started
+To get started you’ll do the following:
+1. Create a Code Builder Environment.
+2. Create a Salesforce DX project.
+3. Connect to your Org by logging in.
+4. Now you’re ready to use Code Builder.
+
+## Connect to Your Dev Environment
+1. From the App Launcher, find and open **Code Builder**.
+(If you don't see Code Builder as an option, or you see an empty screen when you launch it, contact your admin to make sure you have the correct license and permission set assigned to you.)
+2. Start setting up your Code Builder environment. Click “Get Started”.
+3. Click “New Project to create a new Salesforce DX project.
+4. Give your project a name and choose “Standard” for project type.
+
+Following steps involve connecting an Org to your Code Builder environment. You can choose to connect to a Salesforce Org (`login.salesforce.com`) or a Sandbox Org (`test.salesforce.com`).
+
+1. Go through the standard Salesforce login rituals and give Code Builder access to your org. Click **Allow**.
+2. After you’ve successfully connected to your org, click **Next**.
+3. Choose an alias (nickname) for your org. If you work on many orgs, make sure the nickname helps you identify the org quickly in the future. 
+4. Your new Code Builder environment is now available on your dashboard.
+5. Click **Launch** to launch Code Builder in a new tab. 
+6. Now sit back and relax for a few minutes while Code Builder creates and configures your workspace. Code Builder might take a minute (well, currently  x-y minutes) to start up when it initially sets up.
+
+**Tip**: Once you have created your Code Builder environment, just Launch your Code Builder environment from the dashboard at any time, or bookmark it for fast access.
+
+## Let's Take A Quick Tour
+You’re now in your developer environment in Code Builder. Code Builder has VS Code’s rich IDE plus it gives you easy access to cool Salesforce development-specific tools through the Salesforce extension pack. Let’s take a quick tour and get to know the lay of the land. Your screen looks something like this:
+
+![code_builder_ui](../../../images/code_builder_ui.jpg)
+The Code Builder user interface is divided into five main areas.
+
+**Activity Bar**: Located on the left-hand side and it contains iconic buttons to switch between different views. For instance, the Explorer view is active in the above picture.
```

Comment:
```suggestion
**Activity Bar**: Located on the left-hand side, it contains iconic buttons to switch between different views. In our example, the Explorer view is active.
```
Not supposed to use "above" or "below" - at least that's what I was taught. And here's some alternatives in our Style Guide - but I think you can rewrite and skip it as I suggested above. https://confluence.internal.salesforce.com/display/DOCTEAM/below